### PR TITLE
[14.0] [ADD] account_payment_order_inter_company

### PR DESCRIPTION
**Pull request by chafique-delli**

### CODE REVIEW

**Comment by chafique-delli** [comment]
@alexis-via , @sebastienbeau

File: account_payment_order_inter_company/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module allows to map tax and account depending of the product.
```
Comment (rvalyi) [inline account_payment_order_inter_company/readme/DESCRIPTION.rst]: it looks like this is not the proper description. Meanwhile, in the module README.rst, the description looks better:
"This module allows for an inter-company billing to automatically pay the invoice in Company A as soon as the invoice in Company B is paid." So I'm not sure why these two descriptions are different, but it looks this one is not correct...

**Review by rvalyi — CHANGES_REQUESTED** [review]

File: account_payment_order_inter_company/readme/DESCRIPTION.rst (hunk 1 above)
Comment (chafique-delli) [reply account_payment_order_inter_company/readme/DESCRIPTION.rst]: @rvalyi , You're right, I fixed that.

**Comment by chafique-delli** [comment]
@sebastienbeau, @rvalyi

**Comment by legalsylvain** [comment]
@sebastienbeau, @rvalyi : friendly reminder.

**Pull request closed without merging**